src/HTTPBackend.js

```diff
@@ -16,7 +16,7 @@ export default function(options) {
 				makeRequest(url, parsedRequest)
 					.then(resp => parseResponse(parsedRequest, resp))
 					.then(resp => setTimeout(() => messageCallback(JSON.stringify(resp))))
-					.catch(err => messageCallback(JSON.stringify(unknownError(err, parsedRequest))));
+					.catch(err => messageCallback(unknownError(err, parsedRequest)));
```

Comment:
We no longer stringify the error because we want an actual Error object to be passed through